aten/src/THC/THCTensorRandom.cu

```diff
@@ -87,11 +60,6 @@ THC_API __host__ void THCRandom_setRNGState(THCState* state, THByteTensor *rng_s
     THArgCheck(THByteTensor_nElement(rng_state) == total_size, 1, "RNG state is wrong size");
   }
   THArgCheck(THByteTensor_isContiguous(rng_state), 1, "RNG state must be contiguous");
-
-  THCudaCheck(cudaMemcpy(gen->state.gen_states, THByteTensor_data(rng_state),
-                         states_size, cudaMemcpyHostToDevice));
```

Comment:
Ditto here

Reply:
Is this necessary? Since I made all the gen_states memory to have -1 in `getRNGState`, this function will just not affect that value. If I were to do cudaMemcpy or memset here, I need to allocate the gen_states (which I deleted i.e. the initializeGenerator function).